src/wp-includes/widgets.php

```diff
@@ -1099,7 +1099,7 @@ function wp_set_sidebars_widgets( $sidebars_widgets ) {
 		$sidebars_widgets['array_version'] = 3;
 	}
 
-	update_option( 'sidebars_widgets', $sidebars_widgets );
+	update_option( 'sidebars_widgets', $sidebars_widgets, true );
```

Comment:
Better to leave this as is, and handle it in https://core.trac.wordpress.org/ticket/42743 if needed.
```suggestion
	update_option( 'sidebars_widgets', $sidebars_widgets );
```